.prettierrc

```
@@ -0,0 +1,5 @@
+bracketSpacing: true
+trailingComma: 'all'
+useTabs: true
```

I think this is a bit overwhelming. Migrating from no tabs at all to tabs everywhere means lots of changes. +20,000 changes doesn't sound good xd

```suggestion
useTabs: false
```